#include <iostream>
#include "library.hpp"

using namespace std;

const string exceptions::error[] = {"Value is not from the range, results will be wrong.", "Divided by '0' !"};
